projects/continuous_learning/multihead/model.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+Dense/sparse MLP implementation
+"""
+
+import torch
+import torch.nn.functional as F
+
+from nupic.torch.modules import Flatten, KWinners, SparseWeights
+
+
+class Classifier(torch.nn.Module):
```

Comment:
I think we should have a more specific name than `Classsifier`, which could mean almost anything. Maybe `TwoLayerClassifier` or something like that?

Reply:
Sounds good. I also added documentation to the init function.